src/main/java/com/flansmod/common/driveables/EntityDriveable.java

```diff
@@ -237,6 +237,8 @@ public EntityDriveable(World world, DriveableType t, DriveableData d, EntityPlay
         if (owner != null) {
             this.owner = owner;
             ownerUUID = owner.getUniqueID().toString();
```

Comment:
Having cross-referenced 1.12's source, it might be more robust to save a UUID's most and least significant bits instead of as a string.

Reply:
Interesting. The conflict potential could be quite high I imagine?
Would this be in the interest of memory, NBT or?

Reply:
Less chance of a UUID not parsing correctly. It's not that big of a deal though.

Reply:
I'll leave it in